pulp_ansible/app/serializers.py

```diff
@@ -128,10 +129,23 @@ def validate(self, data):
             rest_framework.serializers.ValidationError: If the url or requirements_file is invalid
 
         """
+
+        def _validate_url(url):
+            """Based on https://git.io/JTMAA."""
+            if not url.endswith("/") and url != "https://galaxy.ansible.com":
```

Comment:
maybe we need a constant with URLs, I believe we need to add https://galaxy-dev.ansible.com

Reply:
In order to do that, I'd have to make [this logic](https://github.com/pulp/pulp_ansible/pull/403/files#diff-e6f9543dbfb3c4a3c4c71d8068b87a2388ef384e16d8b19fc61ec7c5e03d7919R344-R345) more complex. I don't think many users will use dev galaxy and if they do, they can use `https://galaxy-dev.ansible.com/api/` which is a link to the actual api.

Reply:
This line is just sort of a nasty hack that I have to carry over from ansible-galaxy. See https://github.com/ansible/ansible/blob/a51a6f4a259b45593c3f803737c6d5d847258a83/lib/ansible/galaxy/api.py#L52-L53.